site/en/docs/devtools/console/javascript/index.md

```diff
@@ -91,6 +93,11 @@ features you're not familiar with. The Console is a perfect place for these kind
     `add(25)` evaluates to `45` because when the `add` function is called without a second argument,
     `b` defaults to `20`.
 
+5.  You will not be able to run any code in this console session until your function has returned.
+    If that takes too long, you can use the **Task Manager** to cancel the time-intensive computation;
+    however, it will also cause the current page to fail and all data you have entered will be lost.
```

Comment:
```suggestion
You will not be able to run any code in this console session until your function has returned.
If that takes too long, you can use the **Task Manager** to cancel the time-intensive computation;
however, it will also cause the current page to fail and all data you have entered will be lost.
```

Reply:
I've removed the number because this is a meta comment about the procedure's result, not an action the reader needs to take.